### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by tcrasset** [EbubekirKocadag/DL-wine-quality#1]
# Structured data review

All the necessary points are there! Good job

## Good

- Very nice accuracy (maybe too much ?)
- Like the resampling
- The training is successful, the loss is decreasing
- I like the README with the conclusion and the steps

## To improve

- Create functions outside of a `.ipynb` file so that you reduce code duplication across different notebooks, e.g. `training`, `compute_score` methods
- You could have done some EDA
- You could have used a validation set
